### ISSUE TITLE
Socialite not extending

### ISSUE THREAD

**Issue by sweebee** [mollie/laravel-mollie#73]
MollieServiceProvider is not loading out of console. I've added a ```dd('test')``` in this file. but nothing shows up if i open the application, but if i run an artisan command it shows up.

## Expected Behavior
Should load provider with socialite extension. because it shows now ```Driver [mollie] not supported.```

## Your Environment
Laravel 5.7.10
Socialite 3.2
Laravel-mollie 2.3.0

**Comment by sandervanhooft** [comment]
Hi @sweebee ,

Thanks for opening this ticket.

Are you sure you are invoking the Mollie services (i.e. using `mollie()`) from the url you are accessing?

Bootstrapping the MollieServiceProvider is [deferred](https://laravel.com/docs/5.7/providers#deferred-providers) (meaning it's lazily loaded).

If you are, can you help me reproduce your issue step by step?

**Comment by sweebee** [comment]
I've got a route ```Route::get('mollie/login', 'MollieController@login');``` which calls this function:

```
public function login()
	{
		return Socialite::with('mollie')
		                ->scopes(['profiles.read profiles.write payments.read payments.write refunds.read refunds.write settlements.read organizations.read invoices.read'])
		                ->redirect();
	}
```

then i get:

```
InvalidArgumentException
Driver [mollie] not supported.
```

**Comment by sandervanhooft** [comment]
> If i add this manually to AppServiceProvider it works, but it should not be neccecery ...

Indeed. If you come up with a permanent fix instead of this workaround, please submit a PR.

Otherwise I'll look into it but it may take a while, kinda busy with Laravel Cashier Mollie atm.

Possibly related issues: https://github.com/mollie/laravel-mollie/issues/5, https://github.com/mollie/laravel-mollie/issues/49.

**Comment by sandervanhooft** [comment]
Could you perhaps check what `dd(interface_exists('Laravel\Socialite\Contracts\Factory'));` yields?

**Comment by sandervanhooft** [comment]
Another fix could be setting `$defer = false` on the `MollieServiceProvider`.

**Comment by sweebee** [comment]
putting ```dd(interface_exists('Laravel\Socialite\Contracts\Factory'));``` in my MollieController returns ```true```.

Setting $defer to false does nothing.

**Comment by sandervanhooft** [comment]
Ok, thanks!

**Comment by sandervanhooft** [comment]
Hi @sweebee,

So far I'm unable to recreate your issue.

You mentioned
> MollieServiceProvider is not loading out of console.

Do you mean you're invoking this from the command line?

**Comment by sandervanhooft** [comment]
I think I've just recreated it.
And then fixed it by setting `$defer` to `false`. AND do a `composer dump-autoload`.

**Comment by sandervanhooft** [comment]
@sweebee Can you check if that works for you?

**Comment by sandervanhooft** [comment]
#76

**Comment by sweebee** [comment]
@sandervanhooft yes its working then.

**Comment by sandervanhooft** [comment]
Hi @sweebee ,

I've just released 2.3.1, which disables the deferred loading. So you should be good to go. Let me know if the issue persists.
